ex1/exceptions.py

```diff
@@ -46,3 +53,10 @@
 # the first by the second. This function should handle exceptions that might
 # occur print out what went wrong and return None if no results could be
 # computed.
+def safe_divide( a, b ):
+    ret = 0
+        try:
```

Comment:
This should not be indented. That's the reason why this module is not tested in travis.